### PR TITLE
Subordinate service block styling

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
### QA
- Add a subordinate charm to the canvas and check the design matches below
- Add a none subordinate charm and relate them
- Check the relation line works as expected
- Check it updated the subordinate relation count

2c30813a-6604-11e5-874f-16d57d401069

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3096/
Test FAILed.

File: jujugui/static/gui/src/app/views/topology/service.js [hunk 1]
```diff
@@ -1592,7 +1627,7 @@ YUI.add('juju-topology-service', function(Y) {
         }
         selection = this.selectionFromServiceNames(serviceNames);
       }
-      selection.classed(topoUtils.getVisibilityClasses('show'));
+      //selection.classed(topoUtils.getVisibilityClasses('show'));
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/topology/service.js]: commented out?

**Comment by hatched** [comment]
:+1: This looks awesome!

QA Notes:
- I noticed that the circle in the middle of the relation line isn't centred when relating to a subordinate but is when relating two services. 
- The small circles for the subordinate and unit count appear to have pretty bad aliasing, is there anything we can do to smooth those out?

screenshot from 2015-09-30 21 09 01
screenshot from 2015-09-30 21 08 37

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3112/
Test PASSed.

**Comment by hatched** [comment]
:+1: QA OK!

Thanks for the updates, looks like there wasn't much you could do about the jittery smaller circles :(

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3117/
Test FAILed.

**Comment by hatched** [comment]
:shipit: Thanks for this fix! Looks great

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3118/
Test FAILed.

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1069

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3119/
Test PASSed.

**Comment by anthonydillon** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**